packages/dashboard/src/customization/propertiesSections/thresholdSettings/thresholdsSection.tsx

```diff
@@ -139,19 +116,16 @@ const ThresholdsSection: FC<ThresholdsSectionProps> = ({
     return null;
   };
 
-  const { onAddNewThreshold, thresholdsComponent } = renderThresholds();
+  const thresholdsComponent = renderThresholds();
   const annotationsComponent = renderAnnotations();
 
   return (
-    <ThresholdsExpandableSection
-      expanded={isExpanded}
-      updateExpanded={setIsExpanded}
-      onAddNewThreshold={onAddNewThreshold}
-      addNewThresholdEnabled={thresholdsEnabled}
-    >
-      {annotationsComponent}
-      {thresholdsComponent}
-    </ThresholdsExpandableSection>
+    <>
+      <ThresholdsExpandableSection title='Thresholds'>{thresholdsComponent}</ThresholdsExpandableSection>
+      {annotationsComponent && (
+        <ThresholdsExpandableSection title='Threshold style'>{annotationsComponent}</ThresholdsExpandableSection>
```

Comment:
The naming should be generic ? given ThresholdsExpandableSection renders both thresholds and annotations ?

Reply:
or may just have something like 
`
[thresholdsComponent, annotationsComponent].map( (something)) => { return something ? <ExpandableSection headerText={<ExpandableSectionHeader>{title}</ExpandableSectionHeader>} defaultExpanded .... : </> }`